src/components/Pollution.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import { faArrowLeft, faMicrophone, faGear} from "@fortawesome/free-solid-svg-icons";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useLocation, useNavigate } from "react-router-dom";
+import { fetchPollutionData,selectPollutionData } from "../redux/pollutionSlice";
+
+const Pollution = () => {
+
+  let location = useLocation()
+  let data = location.state
+  let navigate = useNavigate()
+  const handleGoBackButton = ()=>{
+    navigate(-1)
+  }
+
+  let pollutionData = useSelector(selectPollutionData(data.name))
+  let dispatch = useDispatch()
+
+  useEffect(() => {
+    if(!pollutionData){
+
+      dispatch(fetchPollutionData({country:data.name,lat:data.lat,lon:data.lon}))
+    }
+
+
+  }, [data])
+
+
+  return (
+<>
+{/* <div className = "pollution-card-header"> */}
+<FontAwesomeIcon className = "left-arrow-icon" onClick={handleGoBackButton} icon={faArrowLeft} size="1xs" />
+<FontAwesomeIcon className = "microphone-icon" icon={faMicrophone} size="1xs" />
+<FontAwesomeIcon className = "gear-icon" icon={faGear} size="1xs" />
+{/* <p className = "polution-stats">Pollution stats</p> */}
+{/* </div> */}
+<div className="pollution-card">
+<div className="pollution-details">
+  <h1>{data.name}</h1>
+  <h2>Pollution Data</h2>
+  <p>Air Pollutants</p>
+  <p>no : {pollutionData?.no}</p>
+  <p>co : {pollutionData?.co}</p>
+  <p>no2 : {pollutionData?.no2}</p>
+  <br />
+  <p>Air Quality:  {pollutionData?.stringQuality}</p>
+</div>
+</div>
+</>
+  )
+}
```

Comment:
- [ ] As requested by the previous reviewers, Kindly stick to the given design and make your details page take the full width of the page and fix all elements' placement, widths, and margins to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/227724529-22868fbf-c611-48d4-8fd1-f1fabb58da26.png)  | ![image](https://user-images.githubusercontent.com/68796232/227724512-2c8c6131-f38c-4c25-8144-20cfa8318a3e.png)  |